src/Qir/Tools/Microsoft.Quantum.Qir.Runtime.Tools.csproj

```diff
@@ -36,7 +36,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Quantum.QirGeneration" Version="0.24.217480-beta" />
+	<PackageReference Include="Microsoft.Quantum.QirGeneration" Version="0.25.223055-beta" />
```

Comment:
The indentation of 4 spaces has been replaced with a single tab. You likely didn't mean to do that.

Reply:
fixed